### PR TITLE
Feature/drbd monitoring improvements

### PR DESCRIPTION
**Pull request by arbulu89**
Follow up of https://github.com/SUSE/ha-sap-terraform-deployments/pull/269

- Improve salt state duplications. `cluster_node` salt state created to gather all of the states related to cluster nodes (hana, drbd, netweaver)

- Add drbd and netweaver monitoring jobs to prometheus

- Add some documentation

- Some updates in all `variables.tf` files to remove deprecated options (type value into quotes)

### CODE REVIEW

File: salt/drbd_node/drbdsetup_link.sls [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Workaround to make drbdsetup available among different SLE versions
+/usr/sbin/drbdsetup:
+  file.symlink:
+    - target: /sbin/drbdsetup
```
Comment (stefanotorresi) [inline salt/drbd_node/drbdsetup_link.sls]: this is won't be needed anymore as soon as we release next ha_cluster_exporter version, which will happen before the end of the current week, so I'd suggest to skip it!

Comment (arbulu89) [reply salt/drbd_node/drbdsetup_link.sls]: Removed!

File: libvirt/modules/netweaver_node/salt_provisioner.tf [hunk 2]
```diff
@@ -51,7 +51,9 @@ role: netweaver_node
 netweaver_inst_media: ${var.netweaver_inst_media}
 netweaver_nfs_share: ${var.netweaver_nfs_share}
 ha_sap_deployment_repo: ${var.ha_sap_deployment_repo}
+shared_storage_type: shared-disk
```
Comment (diegoakechi) [inline libvirt/modules/netweaver_node/salt_provisioner.tf]: @arbulu89 Is this hardcoded on purpose?

Comment (arbulu89) [reply libvirt/modules/netweaver_node/salt_provisioner.tf]: Yes...
By now the code is not adapted to use `sbd` with a disk provided by `iscsi` in Netweaver cluster. The shared disk is always created.
This is other improvement that is pending. But the whole iscsi usage needs a big refinement

**Pull request merged**